File: Documentation/API/Parsers/Chart/ReadTimeSignatureChangesFromChartData.cs
using System;
using System.IO;
using RhythmGameUtilities;

var contents = File.ReadAllText("./tests/Mocks/song.chart");

var timeSignatureChanges = Chart.ReadTimeSignatureChangesFromChartData(contents);

Console.WriteLine(timeSignatureChanges.Length); // 4

File: Documentation/API/Parsers/Chart/ReadTimeSignatureChangesFromChartData.js
import { readFileSync } from 'node:fs';

import RhythmGameUtilities from '@rhythm-game-utilities/core';

const contents = readFileSync('./tests/Mocks/song.chart', 'utf-8');

const timeSignatureChanges = RhythmGameUtilities.ReadTimeSignatureChangesFromChartData(contents);

console.log(timeSignatureChanges.length); // 4
